Tasks/Common/securefiles-common/securefiles-common.ts

```diff
@@ -30,8 +30,16 @@ export class SecureFileHelpers {
         const file: NodeJS.WritableStream = fs.createWriteStream(tempDownloadPath);
 
         const agentApi = await this.serverConnection.getTaskAgentApi();
+
+        const ticket = tl.getSecureFileTicket(secureFileId);
+        if (!ticket) {
+            // tl.loc currently only works if the consuming tasks define the resource string.
```

Comment:
That has been the case for a while, the consuming task needs to duplicate the loc messages.

Reply:
What's the vsts-task-lib version of the task consuming it and not showing them correctly?

There was a change [here](https://github.com/Microsoft/vsts-task-lib/pull/295) to allow multiple resource files. That was version 2.2.0 of the task lib.

Reply:
We have an email thread on this, but it seems like the task-lib version of securefiles-common is not being honored by consumers.  So this would change task to task.

Reply:
I filed #7491